pkgs/applications/audio/aeolus/default.nix

```diff
@@ -16,7 +16,7 @@ stdenv.mkDerivation rec {
     libX11 libXft readline
   ];
 
-  patchPhase = ''sed "s@ldconfig.*@@" -i source/Makefile'';
+  patchPhase = ''sed "s@ldconfig.*@@" -i ${src.name}/Makefile'';
```

Comment:
This does not make sense. The default `name` of the FOD produced by `fetchurl` is the basename of the URL, with file name extension included. In this case, it is currently `"aeolus-0.10.4.tar.bz2"`, and the sensible value is `"aeolus-0.10.4"`.

In my opinion, commands in phases should respect `"$sourceRoot"`, so that overriding with `src` from a different fetcher is possible.